src/main/java/duke/Task.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+/**
+ * This class is an object-oriented-representation of a task event
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns statusIcon in the form of 1 or 0.
+     * 1 means done, 0 means undone.
+     *
+     * @return String representation of statusicon.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "1" : "0");
```

Comment:
From the CS2103 Java Coding Standard:

Here are two acceptable ways to format ternary expressions:
```
alpha = (aLongBooleanExpression) ? beta : gamma;
alpha = (aLongBooleanExpression)
        ? beta
        : gamma;
```

Could consider placing the parentheses around the isDone clause rather than the whole return term.